trade-finance-manager-api/src/v1/controllers/tasks.api-test.js

```diff
@@ -391,22 +391,22 @@ describe('tasks controller', () => {
         const nonExistantTaskId = 789;
 
         await expect(
-          updateTfmTask({ dealId: updatableTaskDealId, groupId, taskId: nonExistantTaskId, taskUpdate: updatableTaskUpdateToDo }),
+          updateTfmTask({ dealId: updatableTaskDealId, groupId, taskId: nonExistantTaskId, taskUpdate: updatableTaskUpdateToDo, sessionTfmUser: MOCK_USERS[0] }),
```

Comment:
does your formatter not want this as multiple lines?

Reply:
Oops 😅